src/BaGetter.Web/Pages/Shared/_Layout.cshtml

```diff
@@ -27,6 +30,13 @@
                         <ul class="nav navbar-nav" role="tablist">
                             <li role="presentation"><a asp-area="" asp-page="/Index" role="tab" nav-link><span>Packages</span></a></li>
                             <li role="presentation"><a asp-area="" asp-page="/Upload" role="tab" nav-link><span>Upload</span></a></li>
+                            @{
+                                var options = configuration.Get<BaGetterOptions>();
```

Comment:
Why not directly inject `IOptionsSnapshot<BaGetterOptions>` instead?

Reply:
done